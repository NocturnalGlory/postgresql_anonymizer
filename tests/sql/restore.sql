

CREATE DATABASE contrib_regression_test_restore;

\! psql contrib_regression_test_restore -c 'CREATE EXTENSION anon CASCADE;'
\! psql contrib_regression_test_restore -c 'SELECT anon.start_dynamic_masking();'
\! pg_dump -Fc contrib_regression_test_restore > contrib_regression_test_restore.pgsql

DROP DATABASE contrib_regression_test_restore;

\! pg_restore -d postgres --role=postgres -C contrib_regression_test_restore.pgsql

DROP DATABASE contrib_regression_test_restore;
