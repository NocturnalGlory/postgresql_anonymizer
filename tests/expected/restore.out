CREATE DATABASE contrib_regression_test_restore;
ERROR:  database "contrib_regression_test_restore" already exists
\! psql contrib_regression_test_restore -c 'CREATE EXTENSION anon CASCADE;'
ERROR:  extension "anon" already exists
\! psql contrib_regression_test_restore -c 'SELECT anon.start_dynamic_masking();'
 start_dynamic_masking 
-----------------------
 t
(1 row)

\! pg_dump -Fc contrib_regression_test_restore > contrib_regression_test_restore.pgsql
DROP DATABASE contrib_regression_test_restore;
\! pg_restore -d postgres --role=postgres -C contrib_regression_test_restore.pgsql
DROP DATABASE contrib_regression_test_restore;
